grama/dfply/reshape.py

```diff
@@ -1,14 +1,35 @@
+__all__ = [
+    "tran_arrange",
+    "tf_arrange",
+    "tran_rename",
+    "tf_rename",
+    "tran_separate",
+    "tf_separate",
+    "tran_unite",
+    "tf_unite",
+    "tran_gather",
+    "tf_gather",
+    "tran_spread",
+    "tf_spread",
+    "tran_explode",
+    "tf_explode",
+    "convert_type"
+]
+
 from .base import *
+from .. import add_pipe
 import re
+import numpy as np
```

Comment:
should import only those functions in np and pd that are necessary for this file